### PR TITLE
removed blue border & added hover effect

### PR DESCRIPTION
**Pull request by PradnyaB17**

### CODE REVIEW

File: src/styles/icon.css [hunk 1]
```diff
@@ -15,4 +15,7 @@
   .plusIcon {
     @apply h-full opacity-30;
   }
+  .hamburgerIcon {
+    border: 1px solid rgb(221, 113, 198); /* Set the border width and color */
+  }
```
Comment (anicmarko9) [inline src/styles/icon.css:20]: Please follow TailwindCSS syntax for this. It should be something like this:
```css
.hamburgerIcon {
  @apply border border-primary
}
```
Maybe like that, maybe not, I can't really know on top of my head, but convert basic CSS into TailwindCSS syntax.

**Review by anicmarko9 — COMMENTED** [review]
Thanks for your contribution. I'll wait for the small change before I merge it into `main`.

File: src/styles/icon.css (hunk 1 above)
Comment (anicmarko9) [reply src/styles/icon.css:20]: Also, don't use RGB, but rather use one of the colors from `tailwind.config.js`.

Comment (PradnyaB17) [reply src/styles/icon.css:20]: I've made the change, can you review it once again?
Thank you!

**Review by anicmarko9 — APPROVED** [review]
I suppose this is it... Thanks for your contribution.

**Pull request merged**